src/MSONParameterParser.h

```diff
@@ -178,6 +181,8 @@ namespace snowcrash {
                     }
                     else {
                         if (RegexMatch(attributes[i], "^enum\\[[^][]+]$")) {
+
+                            // Retrieve the type which is wrapped by enum[]
                             out.node.type = attributes[i].substr(5, attributes[i].length() - 6);
```

Comment:
Already made it tolerant to white spaces. Regarding constants, I don't agree with that. We are stripping `enum[` and `]` which are hardcoded and therefore I hardcoded the numbers `5` and `6` just like how we hardcoded some numbers in MSON parsing for parsing some punctuations.